types/pmx/tcr/cat1.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$id": "https://akenza.io/pmx/tcr/cat1.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "cat1",
+  "title": "Filter Cat 1 Values",
+  "properties": {
+    "LTRCounter": {
```

Comment:
usually we are using the following convention: camelCase properties should always start with a lower case character even for acronyms